docker/docker-compose.yml

```diff
@@ -1,38 +1,40 @@
-version: '3.3'
-services:
-
-  zookeeper:
-    image: confluentinc/cp-zookeeper:3.3.0
-    environment:
-      ZOOKEEPER_CLIENT_PORT: '2181'
-
-  kafka:
-    image: confluentinc/cp-kafka:3.3.0
-    ports:
-    - "9093:9093"
-    depends_on:
-    - zookeeper
-    environment:
-      KAFKA_ZOOKEEPER_CONNECT: zookeeper:2181
-      KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR: '1'
-      KAFKA_ADVERTISED_LISTENERS: PLAINTEXT://kafka:9092,OUTSIDE://localhost:9093
-      KAFKA_LISTENER_SECURITY_PROTOCOL_MAP: PLAINTEXT:PLAINTEXT,OUTSIDE:PLAINTEXT
-
-  kms:
-    image: skyuk/kafka-message-scheduler:latest<localBuild>
-    depends_on:
-    - kafka
-    environment:
-      KAFKA_BROKERS: kafka:9092
-      SCHEDULE_TOPICS: schedules
-    ports:
-    - "9095:9095"
-
-  prometheus:
-    image: prom/prometheus:v2.25.2
-    ports:
-      - "9090:9090"
-    volumes:
-      - ./prometheus.yml:/etc/prometheus/prometheus.yml
-    command:
-      - '--config.file=/etc/prometheus/prometheus.yml'
+version: '3.8'

```

Comment:
The one at the root level is designed to work without the local build, the one inside docker/docker-compose.yml is for local development and can be started with sbt-docker-compose. However I'm not sure of it's use as we don't have any integration tests that use the docker image, so would be open to keeping a single docker-compose file at the root level

Reply:
i would probably opt for keeping this one, its easier to use as it will do the local build for you. We probably should add some e2e tests that use it anyway (in the future).